typeDefs.js

```diff
@@ -396,6 +403,7 @@ const typeDefs = gql`
     switchUserStatus(id: ID!): LogicalResult
     resetPassword(password: String!, token: String!): Boolean
     checkIfTokenIsValid(token: String!): Boolean
+    confirmSpecialUser(user: specialUserConfirmInput!,token: String!): LogicalResult
```

Comment:
It is better to rename it to the `completeAdminRegister`

Reply:
fixed